python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -405,14 +411,15 @@ def validate(self):
 
         return (True, '')
 
-    def handleUploadFailure(self, error):
+    def handleUploadFailure(self, error, cmd=''):
```

Comment:
Yes but we now know what command caused the failure. This was frankly impossible before now.